packages/app-project/src/components/ProjectHeader/components/Nav/components/NavLink/NavLink.js

```diff
@@ -1,46 +1,50 @@
 import { SpacedText } from '@zooniverse/react-components'
-import { Anchor, Box } from 'grommet'
-import { shape, string } from 'prop-types'
+import { Anchor } from 'grommet'
+import Link from 'next/link'
+import { withRouter } from 'next/router'
 import React from 'react'
 import styled from 'styled-components'
-import { withRouter } from 'next/router'
 
 const StyledSpacedText = styled(SpacedText)`
   text-shadow: 0 2px 2px rgba(0, 0, 0, 0.22);
 `
 
 const StyledAnchor = styled(Anchor)`
   border-bottom: 3px solid transparent;
-  &:hover,
-  &:focus {
+  &:hover {
     text-decoration: none;
-    border-color: white;
   }
-  ${props =>
-    props.isActive &&
-    `
-    border-color: white;
-  `}
+  &[href]:hover {
```

Comment:
Should be styled for `:focus` too, or are we relying on default styles for focused links?

Reply:
That's my thought (see comment below). FWIW I'm not a fan of the big `box-shadow` Grommet uses by default, but equally I'm not sure if the `border-bottom` alone is enough of a visual signifier. Thoughts?

Reply:
cc'ing @beckyrother - any thoughts on what focus styles should look like?

Reply:
If this isn’t enough of a visual signifier for focus, then it isn’t enough for hover either. Personally I find the default focus outline  very jarring but that’s just my opinion.

Reply:
Could someone post screenshots as examples please? I'm also not a fan of the default focus outline but would like to see what Grommet's looks like before making a decision.

Reply:
This is the default Grommet focus, which is what's being used atm

<img width="277" alt="Screenshot 2019-08-16 18 01 22" src="https://user-images.githubusercontent.com/2725841/63184744-f8c32a00-c04f-11e9-9819-b3b4ab89d370.png">

Reply:
I don't hate this – maybe if it were possible to add some padding around the text it might not feel so weird?

Reply:
If you don't hate it, then let's leave for another day